File: 백준/Bronze/28062. 준석이의 사탕 사기/준석이의 사탕 사기.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');
const N = Number(input[0]);
const candys = input[1].split(' ').map(Number);
const candySum = candys.reduce((acc, cur) => acc + cur, 0);

if (candySum % 2 === 0) {
    console.log(candySum);
} else {
    const oddCandys = candys.filter(c => c % 2 === 1);
    if (oddCandys.length === 0) {
        console.log(0);
    } else {
        console.log(candySum - Math.min(...oddCandys));
    }
}

File: 백준/Bronze/28062. 준석이의 사탕 사기/준석이의 사탕 사기.py
N = int(input())
candys = list(map(int, input().split()))
candy_sum = sum(candys)

if candy_sum % 2 == 0:
    print(candy_sum)
else:
    odd_candys = [c for c in candys if c % 2 == 1]
    if not odd_candys:
        print(0)
    else:
        print(candy_sum - min(odd_candys))